### ISSUE TITLE
macbook proのタッチバーにツイートを表示したい

### ISSUE THREAD

**Issue by yimajo** [yimajo/MeetupTweet#6]

**Comment by takasek** [comment]
https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/OSXHIGuidelines/AbouttheTouchBar.html

ガイドラインによると

>Use the Touch Bar as an extension of the keyboard and trackpad, not as a display.

とのことなので、どうでしょう…？
💭 でも、真面目にガイドラインに従うアプリってわけでもなかったりしますか？

**Comment by yimajo** [comment]
@takasek はい。その通りで真面目に従うアプリってわけではないですね。申請するわけでもないのと、もし使いたい人がいれば勝手に改良してもらえれば良いというのが理由です（そもそもツイートがウインドウに流れること自体がヒューマンインターフェースとして逸脱しているとも思いますw）。

**Comment by takasek** [comment]
>そもそもツイートがウインドウに流れること自体がヒューマンインターフェースとして逸脱しているとも思いますw

確かにそもそもでしたねｗ
